content/altinn-studio/guides/development/prefill/query-params/_index.en.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Prefilling data based on query parameters
+linktitle: Query parameters
+description: How to configure prefill for an app based on query parameters.
+toc: false
+weight: 400
+---
+
+Altinn apps support prefill based on query parameters.
```

Comment:
Should there be a disclaimer on the top here, letting you know which version(s) are needed for this to work?